telos-backend/src/routes/api/journal.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+
+router.get('/', (req, res) => {
```

Comment:
Whats the intention of this endpoint, is it about pages in the journal? Don't disagree, but just wondering as it is different from the widget based ones.

Reply:
My interpretation is that we can GET all widgets associated with the journal for a certain date/range